contribute/components/links.md

```diff
@@ -28,12 +27,10 @@ An example of a link to an auto-generated reference page that we pull in during
 ## HTML
 
 ```html
-
-It is best practice if [a link opens in a new window](https://docker.com/){: target="_blank" rel="noopener" class="_" }
+It is best practice if [a link to an internal page opens in the same window](/) and [a link to an external page opens in a new window](https://docker.com/){: target="_blank" rel="noopener" class="_" }
 
 You can also have [a markdown link to a custom target ID](#formatting-examples)
```

Comment:
Would it be possible to include internal link best practices here? I've seen both internal links to the path (`/subscription/add-seats/`) and to the markdown file (`../docker-id/index.md`). What is the preferred format? Do we want to callout the best practice to add `/` to the end of the link?

Reply:
@stephaurelio Updated somewhat. While on links I also noticed with

`{: target="_blank" rel="noopener" class="_" }` apparently `rel="noopener"` is no longer needed and as far as I can tell looking at HTML, `class="_"` doesn't do anything.